frontend/components/interests/form/InterestForm.tsx

```diff
@@ -79,6 +79,12 @@ export default function InterestForm({ onSubmit }: InterestFormProps) {
                   Your Campaign
                 </Typography>
 
+                <InterestFormCharitySelector
+                  name="charities"
+                  label="Charities supported"
+                  placeholder="Choose your campaign beneficiaries."
+                />
+
                 <InterestFormTextInput name="campaignName" label="Name" placeholder="Give your campaign a name." />
```

Comment:
```suggestion
                <InterestFormTextInput name="campaignName" label="Campaign Name" placeholder="Give your campaign a name." />
```

One more thing, when I tested it, it was giving me name (siva, bharath, etc) autocomplete suggestions. May want to turn that off.

Reply:
Tracked in #135